core/src/main/java/org/mule/runtime/core/privileged/processor/chain/AbstractMessageProcessorChain.java

```diff
@@ -200,7 +205,7 @@ public CoreEvent process(CoreEvent event) throws MuleException {
                                             }))
                                                 .map(result -> result.reduce(me -> {
```

Comment:
I think that this duplicates MessageProcessors.propagateErrorResponseMapper() -wich btw doesn't have the identity() sugar-